css/properties/text-align.json

```diff
@@ -263,6 +263,57 @@
               "deprecated": false
             }
           }
+        },
+        "string": {
+          "__compat": {
+            "description": "Character-based alignment in a table column (<code>&lt;string&gt;</code> value)",
+            "support": {
+              "webview_android": {
```

Comment:
Would you mind sorting this alphabetically (in other words, move WebView to the end)?

Reply:
Of course, done 👍